packages/terminal-extension/src/index.ts

```diff
@@ -179,5 +179,8 @@ function activate(app: JupyterLab, services: IServiceManager, mainMenu: IMainMen
     });
   }
 
+  app.contextMenu.addItem({command: CommandIDs.createNew, selector: '.jp-TerminalWidget', rank: 0});
```

Comment:
We are working on other UI/UX approaches for creating new activities. Because of this, I think we should keep the "New *" actions out of these context menus.